code/modules/surgery/organs/organ_external.dm

```diff
@@ -680,6 +684,17 @@ Note that amputating the affected organ does in fact remove the infection from t
 		tbrute = 3
 	return "[tbrute][tburn]"
 
+/obj/item/organ/external/proc/update_splints() //we're processing splints like this instead of on a per step basis for efficiency
```

Comment:
Mhm, I thought about that, I thought I could get it by without a list of splinted organs but since I ended up having to use one anyways I'm switching it over to a per step basis.